common/lib/xmodule/xmodule/partitions/partitions_service.py

```diff
@@ -74,7 +74,7 @@ def _create_enrollment_track_partition(course):
 
     partition = enrollment_track_scheme.create_user_partition(
         id=ENROLLMENT_TRACK_PARTITION_ID,
-        name=_(u"Enrollment Tracks"),
+        name=_(u"Enrollment Track Groups"),
```

Comment:
@catong Note that this will impact what is shown in the visibility dialog as well.